<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QPushButton" name="btnCapture">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="containersFrame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>171</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brillo</string>
     </property>
    </widget>
    <widget class="QSlider" name="tbSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbFlip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Flip 90 H</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFace">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>FaceDetector</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCapture</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ClickBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>SliderChangeValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFace</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnFaceDectector()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFlip</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>FlipImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedIndexCom()</slot>
  <slot>selectedIndexComString()</slot>
  <slot>ClickBtn()</slot>
  <slot>SliderChangeValue()</slot>
  <slot>BtnFaceDectector()</slot>
  <slot>FlipImage()</slot>
 </slots>
</ui>
